web/controllers/caseload_controller.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Healthlocker.CaseloadController do
+  use Healthlocker.Web, :controller
+
+  alias Healthlocker.{EPJSTeamMember, EPJSUser, ReadOnlyRepo, User}
+
+  def index(conn, _params) do
```

Comment:
All the looping and multiple queries are going to cause this action to be quite slow, but it's also going to get slower with time as we have more records.

Once we get the actual data let's look at refactoring.